buildSrc/src/main/kotlin/buildsrc/config/publishing.kt

```diff
@@ -0,0 +1,70 @@
+package buildsrc.config
+
+import org.gradle.api.Action
+import org.gradle.api.Project
+import org.gradle.api.artifacts.repositories.PasswordCredentials
+import org.gradle.api.provider.Provider
+import org.gradle.api.provider.ProviderFactory
+import org.gradle.api.publish.maven.MavenPom
+import org.gradle.api.publish.maven.MavenPublication
+import org.gradle.kotlin.dsl.findByType
+import org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension
+import org.jetbrains.kotlin.gradle.targets.jvm.KotlinJvmTarget
+
+fun MavenPublication.createSkrapeItPom(
+    configure: MavenPom.() -> Unit = {}
+): Unit = pom {
+    name.set("skrape{it}")
+    description.set("A Kotlin-based testing/scraping/parsing library providing the ability to analyze and extract data from HTML (server & client-side rendered). It places particular emphasis on ease of use and a high level of readability by providing an intuitive DSL. First and foremost it aims to be a testing lib, but it can also be used to scrape websites in a convenient fashion.")
```

Comment:
small notes:

Now the POM name includes the subproject name.

By default Gradle will use `project.description` for the POM description, so you might want to move this description to the root project, and make the subproject descriptions more specific